README.md

```diff
@@ -9,6 +9,14 @@ This monorepo contains the following packages:
 
 It also contains the documentation / demo app (see `docs/`)
 
+## purpose
+
+Friends don’t let friends ship inaccessible code! These accessible packages include everything from typography and colors, to components like custom form controls. The design and interactions shown throughout this site are intended to show how Deque provides accessible experiences for the users of our own products - through the use of common, accessible components like these.
+
+## usage
+
+Cauldron React is an accessible React components library. Cauldron Styles contains accessible styling for those components and more.
```

Comment:
Nit: we could probably link to the respective `package/*` of _Cauldron React_ and _Cauldron Styles_ like we do above, but no big deal.

Reply:
Update pushed!